README.md

```diff
@@ -134,19 +124,25 @@ For more information on the Redis/RabbitMQ requirements, see [#90](/../../issues
 
 1. Create a `.env` file: `cp .env.example .env`
 
-2. Fill in the required information using the output data from `bridgeDeploymentResults.json`. Check the tables with the [set of parameters](#configuration-parameters) below to see their explanation.
+2. Fill in the required information using the JSON output data. Check the tables with the [set of parameters](#configuration-parameters) below to see their explanation. 
 
 ## Run the Processes
 
-There are two options to run the nodejs oracle:
-1. Docker containers. This requires [Docker](https://docs.docker.com/install/) and [Docker Compose](https://docs.docker.com/compose/install/) installed. If you are on Linux, it's also recommended that you [create a docker group and add your user to it](https://docs.docker.com/install/linux/linux-postinstall/), so that you can use the CLI without sudo.
+There are two options to run the TokenBridge processes:
+1. Docker containers. This requires [Docker](https://docs.docker.com/install/) and [Docker Compose](https://docs.docker.com/compose/install/). If you are on Linux, it's also recommended that you [create a docker group and add your user to it](https://docs.docker.com/install/linux/linux-postinstall/), so that you can use the CLI without sudo.
 2. NodeJs Package Manager (NPM).
 
 ### Docker
 
   - While running the bridge containers for the first time use `VALIDATOR_ADDRESS=<validator address> VALIDATOR_ADDRESS_PRIVATE_KEY=<validator address private key> docker-compose up -d --build` 
   - For further launches use `VALIDATOR_ADDRESS=<validator address>  VALIDATOR_ADDRESS_PRIVATE_KEY=<validator address private key> docker-compose  up  --detach`
-  - If you want to use any command from this document, prefix it with `docker-compose exec bridge_affirmation`, if not already prefixed, to execute command inside on of the running docker containers. Make sure bridge service is started before using the commands.
+  - To run additional commands,  use the prefix `docker-compose exec bridge_affirmation` to execute the command inside of the running Docker containers. Make sure the bridge service is started before using the commands. 
+
+     - `docker-compose exec bridge_affirmation npm run watcher:signature-request`
```

Comment:
We need to confirm with @ArseniiPetrovich if this still actual. My understanding is that all workers now can be started by one command.

Reply:
ok. I'm also wondering if we should link to https://github.com/poanetwork/deployment-bridge and include any instructions as an alternative method of setting up and running processes.

Reply:
Yes, please add a line that `deployment-bridge` is preferable way to deploy the bridge instance on the production.

Reply:
Hi, @andorgo
Just checked your PR and can confirm that commands related to services start are obsolete now since services were included into the docker-compose file and launches when the bridge service start.